src/simd_128.h

```diff
@@ -57,7 +57,7 @@ const am128i F3minus1_m128i = _mm_set1_epi32(256);
 static inline aint128 m128i_to_uint128(am128i v)
 {
     aint128 i;
-    _mm_storeu_si128((am128i*)&i, v);
+    _mm_store_si128((am128i*)&i, v);
```

Comment:
Do we have performance improvment thanks to this :slightly_smiling_face: ?
If so, how much?

Reply:
Yes, it improves but not much. But we should work on aligned memory for long term.